server/routes.js

```diff
@@ -401,6 +402,8 @@ export default async app => {
    */
   app.get('/docs/search', gitbook.search);
 
+  app.get('/files/:base64UrlEncodedUrl', filesController.getFile);
+
```

Comment:
Maybe move this under https://github.com/opencollective/opencollective-api/blob/16f30b945fd43d327b5c41e3d9a5d82d4c096812/server/routes.js#L397, in the "File downloads" section